spec/unit/plugin_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'shared_contexts'
+require '3scale_toolbox/cli'
+
+RSpec::Matchers.define_negated_matcher :not_raise_error, :raise_error
+
+RSpec.describe 'Plugin command' do
+  include_context :temp_dir
+  include_context :plugin
+  include_context :random_name
+
+  before(:each) do
```

Comment:
Why not `around` ?

Reply:
it's exactly the same

Reply:
Maybe implementation wise, but definitely not readability wise. 
Around tells the reader it is one logical operation done around the test. This is two independent operations.